### DISCUSSION TITLE
GeneratePendingTransforms should ignore transformations with errors

### DISCUSSION THREAD

**Discussion by kringkaste** [craftcms/cms#11970] [Ideas] [answered]
When a transformation fails and the table row has error = 1, the `GeneratePendingTransforms` job should not retry to make the transformation every time it runs. The job should ignore these or have an error counter like Laravel has. So you can stop trying after three times or so.

**Comment by brandonkelly** [comment, marked as answer]
Good call. We’ve made that change for Craft 4.3.

**Reply by brandonkelly** [reply]
Craft 4.3 was released earlier this week with this change!
